sw/device/lib/testing/sram_ctrl_testutils.h

```diff
@@ -20,35 +20,43 @@
 typedef uint32_t (*sram_ctrl_testutils_exec_function_t)(volatile uint32_t,
                                                         volatile uint32_t);
 
+/**
+ * Test buffer size in words and bytes.
+ *
+ * Must be multiple of sizeof(uint32_t).
+ */
+#define SRAM_CTRL_TESTUTILS_DATA_NUM_BYTES 16
```

Comment:
nit: it would be more standard to do this the other way round (i.e. declare NUM_WORDS and then NUM_BYTES is just NUM_WORDS  * 4). That way the number of bytes is guaranteed to be a multiple of 4.

Reply:
Not to mention that we aught to use `sizeof(uint32_t)` instead of `4` to be consistent with other places in the code.

Reply:
Done.